doc/imported_samples/marking_criteria.yml

```diff
@@ -0,0 +1,56 @@
+Quality of Writing:
+    type: Rubric
```

Comment:
Use lowercase words for the type (so `rubric` instead of `Rubric`). Same applies to `Checkbox` below.